src/go/k8s/tests/e2e/centralized-configuration-upgrade/01-first-upgrade.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: centralized-configuration-upgrade
 spec:
   image: "vectorized/redpanda"
-  version: "v21.11.16"
+  version: "v22.1.10"
```

Comment:
I assume this test now runs with the new feature gate enabled, right?

Reply:
Yes as we will sunset 21.11.X soon